src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Services/AssemblyLoadContextWrapper.cs

```diff
@@ -52,15 +45,18 @@ public static bool TryLoadExtension(string assemblyFilePath, ILogger? logger, [N
             return false;
         }
 
-        public static AssemblyLoadContextWrapper? TryCreate(string name, string assembliesDirectoryPath, ILogger? logger)
+        public static AssemblyLoadContextWrapper? TryCreate(string name, string assembliesDirectoryPath, ILogger logger)
         {
             try
             {
-                var loadContext = CreateLoadContext(name);
+                logger.LogTrace("[{name}] Loading assemblies in {assembliesDirectoryPath}", name, assembliesDirectoryPath);
```

Comment:
Didn't want to just use interpolated strings?

Reply:
theres actually a diagnostic if you do that.  See https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2254
I noticed while updating this code